src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -86,6 +90,17 @@ class MarathonModule(conf: MarathonConf, http: HttpConf)
     bind(classOf[HttpEventStreamActorMetrics]).in(Scopes.SINGLETON)
   }
 
+  @Named(ModuleNames.SCHEDULER_HEARTBEAT_MONITOR)
```

Comment:
yes exactly. the existing scheduler/guice wiring is complicated and i didn't want to unravel all of that in this PR, especially since one of the goals is to make this easy to backport to earlier releases.